docs/security/trust/trust_delegation.md

```diff
@@ -0,0 +1,199 @@
+local Content Trust data is, and `snapshot -r` specifies that you want to rotate
+the snapshot key specifically, and you want the server to manage it.
+
+Assuming that the file containing the collaborator's public key is named
+`pubkey.pem` (or substitute with a file containing the PEM-encoded x509
+certificate containing a public key)
+
+When adding a delegation, your must acquire
+[the PEM-encoded x509 certificate with the public key](#generating-delegation-keys)
+of the collaborator you wish to delegate to.
+
+Assuming you have the certificate `delegation.crt`, you can add a delegation
+for this user and then publish the delegation change:
+
+```
+$ notary -d ~/.docker/trust delegation add docker.io/<username>/<image> targets/releases delegation.crt --all-paths
+$ notary -d ~/.docker/trust publish docker.io/<username>/<image>
+```
+
+The preceding example illustrates a request to add the delegation
+`targets/releases` to the image repository, if it doesn't exist.  Be sure to use
+`targets/releases` - Notary supports multiple delegation roles, so if you mistype
+the delegation name, the Notary CLI will not error.  However, Docker Engine
+supports reading only from `targets/releases`.
+
+It also adds the collaborator's public key to the delegation, enabling them to sign
+the `targets/releases` delegation so long as they have the private key corresponding
+to this public key.  The `--all-paths` flags tells Notary not to restrict the tag
+names that can be signed into `targets/releases`, which we highly recommended for
+`targets/releases`.
+
+Publishing the changes tells the server about the changes to the `targets/releases`
+delegation.
+
+After publishing, view the delegation information to ensure that you correctly added
+the keys to `targets/releases`:
+
+```
+$ notary -d ~/.docker/trust delegation list docker.io/<username>/<image>
+
+      ROLE               PATHS                                   KEY IDS                                THRESHOLD
+---------------------------------------------------------------------------------------------------------------
+  targets/releases   "" <all paths>  729c7094a8210fd1e780e7b17b7bb55c9a28a48b871b07f65d97baf93898523a   1
+```
+
+You can see the `targets/releases` with its paths and the key ID you just added.
+
+Notary currently has no good way to map collaborators to keys, so we recommend
+that you add and list delegation keys one at a time, and keep a mapping of the key
+IDs to collaborators yourself, should you need to remove a collaborator.
+
+## Removing a delegation from an existing repository
+
+To revoke a collaborator's permission to sign tags for your image, you must know
+the IDs of their keys, because you need to remove their keys from the
+`targets/releases` delegation.
+
+```
+$ notary -d ~/.docker/trust delegation remove docker.io/<username>/<image> targets/releases 729c7094a8210fd1e780e7b17b7bb55c9a28a48b871b07f65d97baf93898523a
```

Comment:
List delegations is above this at the end of adding a delegation.  I'm not sure that listing delegations again before removing would be helpful, since you can't tell which key is which